chart/templates/_helpers.yaml

```diff
@@ -238,21 +238,21 @@ If release name contains chart name it will be used as a full name.
         secretKeyRef:
           name: {{ .Values.dags.gitSync.credentialsSecret | quote }}
           key: GIT_SYNC_USERNAME
-    - name: GITSYNC_USERNAME
+    - name: GIT_SYNC_PASSWORD
       valueFrom:
         secretKeyRef:
           name: {{ .Values.dags.gitSync.credentialsSecret | quote }}
-          key: GITSYNC_USERNAME
```

Comment:
We at least need to add a newsfragment that these will no longer work and it'll use `GIT_SYNC_X` instead.

Reply:
I agree. But I have no idea where and how to put that. Feel free to add/suggest something.

Reply:
just a question, what you think about a solution like this

```{{- $gitUsernameKey := if (hasKey .Values.dags.gitSync.credentialsSecret "GITSYNC_USERNAME") "GITSYNC_USERNAME" "GIT_SYNC_USERNAME" -}}```

Reply:
Good idea, but it's in the secret, not in values so that won't work :(

@tobimichael96, you can read about [how to do a newsfragment over in the contributing docs](https://github.com/apache/airflow/blob/0f38be1f957d72dd8b6081e7f2381f82f513f78a/contributing-docs/16_contribution_workflow.rst?plain=1#L185).

Reply:
I added the fragment, let me know if that works for you.